_content/blog/Tampere.md

```diff
@@ -16,9 +16,9 @@ The City of Tampere is one of Oskari's longest-standing users. The City of Tampe
 
 The user IDs of city officials have been integrated into the Oskari installation through role-based management. In this way, data integrity is maintained. The right people are able to edit and view the datasets they are authorised to edit and view.
 
-Tampere provides and maintains a wide range of substantive data in a geospatial format. Through role-based management, this information can also be viewed by our various stakeholders through our map service. More than 1000 different map layers are available for internal use, of which 500 are publicly available.
+Tampere provides and maintains a wide range of substantive data in a geospatial format. Through role-based management, this information can also be viewed by their various stakeholders through the map service. More than 1000 different map layers are available for internal use, with about 500 being publicly available.
 
-The City of Tampere was one of the organisations that developed the hierarchical layer list function in Oskar. This development is one good example of how we can create features in Oskar that others are also interested in. The hierarchical layer is now part of Oskar's community packages.
+The City of Tampere was one of the organisations that developed the hierarchical layer list function in Oskari. The hierarchical layer is now part of Oskari's community packages.

```

Comment:
Not really part of the "community packages". The original implementation has been deprecated and removed from oskari-frontend, but the current layerlist bundle includes this (and the database structure that makes it possible has been part of Oskari since this was implemented).
```suggestion
The City of Tampere was one of the organisations that developed support for hierarchical structures for layer listing functionality in Oskari. The hierarchical structure of layers have been available for all Oskari users since and has been included in the more recent implementation of the layer listing user interface as well.
```